assessment.sql

```diff
@@ -5,24 +5,46 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees(
+surname CHARACTER VARYING,
+given_name CHARACTER VARYING,
+gender CHARACTER VARYING(1),
+height integer,
+weight integer,
+dob DATE
+eye_color CHARACTER VARYING,
+hair_color CHARACTER VARYING
+)
+
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
+INSERT INTO licensees (surname,given_name,gender,height,weight,dob,eye_color,hair_color)
+VALUES ('Bishop', 'Caren', 'f', 63, 132, 1943-09-26, 'Brown', 'Black');
 
--- your code here
 
 -- Question 3. Load the data from `data/licensees.csv` into the `licensees` table.
 
--- your code here
+\copy licensees(surname,given_name,gender,height,weight,dob,eye_color,hair_color) FROM `data/licensees.csv` WITH (HEADER true, FORMAT csv)
 
 -- Question 4. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
 
--- your code here
+SELECT COUNT (*), eye_color, hair_color
+FROM licensees
+GROUP BY eye_color, hair_color
+;
 
 -- Question 5. Make each of Marilynn Escobar and Chris Whaley one inch shorter. Look them up by their names.
 
--- your code here
+UPDATE licensees
+SET height = height - 1
+WHERE given_name LIKE 'Marilynn', 'Chris'
+WHERE surname LIKE 'Escobar', 'Whaley'
+;
+
 
 -- Question 6. Remove Dylan Rich and Teresita Myers from the table of licensees.
 
--- your code here
+DELETE FROM licensees
+WHERE surname 'Rich', 'Myers'
+WHERE given_name 'Dylan', 'Teresita'
+;
```

Comment:
This does not compare the column values with the string literals at all. It will throw an error for incorrect syntax. This is like writing a sentence where one noun follows another with nothing to indicate how they are or should be related.

The bigger-picture bits are good, though: DELETE is the right statement type, and WHERE clauses will be used to limit the effect of the DELETE to specific rows. 1/4